sdl2_mixer/VITABUILD

```diff
@@ -1,33 +1,40 @@
 pkgname=sdl2_mixer
 pkgver=2.6.3
-pkgrel=1
+pkgrel=2
+gitrev=eb005518b0085244630ccfb41977f818af45d2ea
 url="https://github.com/libsdl-org/SDL_mixer"
-source=("https://www.libsdl.org/projects/SDL_mixer/release/SDL2_mixer-${pkgver}.tar.gz"
-        "pkg-config-fix.patch")
+source=("https://github.com/libsdl-org/SDL_mixer/archive/${gitrev}.tar.gz"
+        "pkg-config-fix.patch"
+        "cmake.patch"
+        )
 sha256sums=(
-    "7a6ba86a478648ce617e3a5e9277181bc67f7ce9876605eea6affd4a0d6eea8f"
-    "9de77978b881b0ab61aa60673ec7de92e920f8affd42ab129d8fe153e2698de5"
+    "SKIP"
```

Comment:
You might want to either include a checksum here or use `git+https://github.com/libsdl-org/SDL_mixer.git#commit=${gitrev}` as source to use git.

Reply:
github archives aren't guaranteed to have same checksum

Reply:
Fair, will add a suggestion then.